scripts/bundleRelease.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
```

Comment:
Question: any reason why you opted for bash instead of Fastlane for this?

Reply:
Most of this bash script already existed - my thinking was that it wouldn't be very much effort to make it generic and this would lead to fewest breaking changes